pkg/registry/registry.go

```diff
@@ -1,38 +1,62 @@
-// Package registry implements a docker V2 registry.
+// Package registry implements a docker V2 registry and the OCI distribution specification.
+//
+// It is designed to be used anywhere a low dependency container registry is needed, with an
+// initial focus on tests.
+//
+// Its goal is to be standards compliant and its strictness will increase over time.
+
 package registry
```

Comment:
Should this be `pkg/v1/registry`? 🤔

Reply:
I don't think so? The go package field is the leaf element generally (i.e. v1 uses package v1).

The language spec indicates only letters and unicode digits are allowed here.
https://golang.org/ref/spec#Package_clause